package.json

```diff
@@ -16,7 +16,7 @@
   },
   "license": "ISC",
   "dependencies": {
-    "@aeternity/aepp-sdk": "^7.5.0",
+    "@aeternity/aepp-sdk": "github:aeternity/aepp-sdk-js#release/7.7.0",
```

Comment:
not released to npm yet?

Reply:
Yeah, PR still not merged. https://github.com/aeternity/aepp-sdk-js/pull/1067